CONTRIBUTING.md

```diff
@@ -21,8 +21,136 @@ Thanks for being willing to contribute to our docs!
 > all of your pull request branches based on this `main` branch. Whenever you
 > want to update your version of `main`, do a regular `git pull`.
 
-## Help needed
+## Written in MDX
 
-Please checkout the [the open issues](https://github.com/clerkinc/clerk-docs/issues)
+Clerk's documentation content is written in variation of markdown called [MDX](https://mdxjs.com/). MDX allows us to embed React components in the content, unlocking rich, interactive documentation experiences. Clerk's documentation site supports [GitHub Flavored Markdown](https://github.github.com/gfm/), adding support for things like tables and task lists.
 
-Also, please watch the repo and respond to questions/bug reports/feature requests! Thanks!
+## Previewing your changes
+
+At this time, we do not currently have way to preview your changes. If you use VSCode, consider using [the built-in markdown preview](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview) to see what the rendered content will look like. Note that this will not include custom components or styles.
```

Comment:
```suggestion
At this time, we do not currently have a way to preview your changes. If you use VSCode, consider using [the built-in markdown preview](https://code.visualstudio.com/docs/languages/markdown#_markdown-preview) to see what the rendered content will look like. Note that this will not include custom components or styles.
```